docs/databases/connections/athena.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Amazon Athena
+---
+
+# Amazon Athena
+
+Connecting Metabase to Athena depends on where Metabase is running.
+
+## Connecting to Athena from a Metabase Cloud instance
+
+If you're running Metabase Cloud, you'll need to upload your IAM credentials:
```

Comment:
Upload suggests that the user needs to upload a file. In this case, it is just writing their credentials like user and password